extensions/adobe/experience/decisioning/decisionevent.schema.json

```diff
@@ -20,7 +20,40 @@
   "description": "A decision event is used to capture observations about the outcome and context of a decision activity. The decision event captures information about how the decision made, when it was occurring, what options were proposed (chosen) and what contextual state existed that either influenced the decision or could be observed during the decision process. The decision event also captures the proposition ID, a globally unique identifier that can be used to correlate the decision to other events. DecisionEvents are not only relatable to Experience Events that impacted a decision but also to ExperienceEvents that are a direct response to a proposition. It is the expectation that applications reference the proposition ID in every experience event that was influenced by the propositions. A user profile's proposition-response history is maintained with these proposition IDs.",
   "definitions": {
     "decisionevent": {
-      "properties": {}
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositions": {
```

Comment:
Directly define the central property in the Decision Event Schema instead of assuming that a mix-in is used. The property is still not required but it defines what makes an event a decision event. Without that property the class schema looses its utility.